### PR TITLE
adding handling for empty dataset to sitevisitpeak

### PR DESCRIPTION
**Pull request by mhines-usgs**
@thongsav-usgs working locally with the empty site visit peak example JSON file

### CODE REVIEW

**Comment by mhines-usgs** [comment]
image

**Comment by mhines-usgs** [comment]
image

File: R/sitevisitpeak-data.R [hunk 1]
```diff
@@ -14,88 +14,100 @@
...
+                     "Date",
+                     "Time",
+                     "Difference from Peak Verification Reading")
+    
+    #Sends in list of readings, and gets pack the formatted data.frame
+    results <- formatSVPData(data$readings,columnNames)
+    
+    return(results)
+  }
+  if (length(data)==0) {
```
Comment (thongsav-usgs) [inline R/sitevisitpeak-data.R]: else statement instead?

File: R/sitevisitpeak-data.R [hunk 2]
```diff
@@ -14,88 +14,100 @@
...
-              nullMask(listElements$associatedIvValue),
-              quals,
-              ivDate,
-              ivTimeFormatting,
-              diff)
-    
-    toRet <- rbind(toRet, data.frame(t(toAdd),stringsAsFactors = FALSE))
+    return(toRet)
+  }  
+  if(length(data) == 0) {
```
Comment (thongsav-usgs) [inline R/sitevisitpeak-data.R]: same comment, use else statement here

Comment (mhines-usgs) [inline R/sitevisitpeak-data.R]: on else it complains and won't build. but else if works.. is that acceptable or should I see why the else alone is failing? @thongsav-usgs

**Pull request merged**